src/infrastructure/service/NetworkServiceAxios.ts

```diff
@@ -25,7 +25,7 @@ export class NetworkServiceAxios implements NetworkService {
         url: `${this.targetUrl}${request.url}`,
         headers: request.headers,
         method: request.method,
-        transformResponse: (data: string) => data,
+        responseType: 'arraybuffer',
```

Comment:
`axiosResponse.data` is typed as `any`, it actually is a `Buffer` but is used as a `string`. Typings should be fixed